source/documentation/continuous-integration-and-delivery/secrets.md

```diff
@@ -0,0 +1,57 @@
+## Secrets
+
+The Reliability Engineering Concourse delegates secrets management to AWS
+Systems Manager Parameter Store and Key Management Service.
+
+Secrets can be managed by users with the `member` role using the [GDS
```

Comment:
This is now the suggested way rather than using the hijack container?

Reply:
They are, in fact, the same thing (see @vixus0's pull request: https://github.com/alphagov/gds-cli/pull/148)

Notify's credentials set up with the GPG key is probably a better approach but requires more initial set up - hopefully we can roll out Notify's approach further)